lock.json

```diff
@@ -1,20 +1,32 @@
 {
   "locks": [
     {
-      "locator": "github.com/ponylang/crypto.git",
+      "locator": "github.com\\ponylang\\crypto.git",
```

Comment:
This looks to be incorrect.

Reply:
Hmm, this would be a bug in corral. Fixed.

Reply:
Can you open an issue against corral?